contentScript.js

```diff
@@ -114,16 +114,21 @@ if (window.location.href.indexOf("bloombergquint.com") !== -1) {
 
 if (window.location.href.indexOf("bloomberg.com") !== -1) {
     document.addEventListener('DOMContentLoaded', () => {
+		const fence = document.querySelector('.fence-body');
+		if (fence)
```

Comment:
To maintain readability you should use one of these examples:

```suggestion
		if (fence) fence.classList.remove('fence-body');
```

or

```suggestion
		if (fence) {
		    fence.classList.remove('fence-body');
		}
```